### ISSUE TITLE
Activating account does not work? 

### ISSUE THREAD

**Issue by Mark-H** [modmore/CommerceTheme_Red#5]
https://secure.helpscout.net/conversation/1266975336/23978?folderId=46284

**Comment by muzzwood** [comment]
On submitting the registration form, user is not redirected and the following shows in log:
```
[2022-04-11 12:41:58] (ERROR @ /www/core/model/modx/modx.class.php : 1037) `[[++ctred.registration_please_activate_page_id]]` is not a valid integer and may not be passed to makeUrl()
[2022-04-11 12:41:58] (ERROR @ /www/core/model/modx/modresponse.class.php : 210) Attempted to redirect to an empty URL.
```
Account is created and email sent as expected however.

**Comment by muzzwood** [comment]
Clicking the link in the email activates the account as expected too.

From initial glance, it just looks to be a mismatch between setting names i.e.
`ctred.registration_activation_page_id` vs `ctred.registration_please_activate_page_id`

**Comment by muzzwood** [comment]
Turned out just be missing setting, resource and template for the registration form to redirect to.
Fixed in commit: [bf4cc92e](https://github.com/modmore/CommerceTheme_Red/commit/bf4cc92ea5a35bd017164e07525c343a581af30d)
